test/rules_test.js

```diff
@@ -32,6 +32,12 @@ describe('rules', () => {
         ['x^(--(x + 1))', 'x^(x + 1)']
     ])
 
+    suite('rearrange coefficient', rules.REARRANGE_COEFF, [
+        ['y^3 * 5', '5 y^3'],
+        ['yz * 3', '3 yz'],
+        ['3x^2 * 5', '5 (3 x^2)']
```

Comment:
Eventually it would be nice to handle `3 * x * 5`, but I don't know what we should do with it.  Maybe ping your content person about that case.  I'm okay with deferring the handle of that case for now.